src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,114 @@
+import FetchRocketsData from '../API/rocketAPI';
+
+const GET_ROCKETS_REQUEST = 'space-travellers/rockets/GET_REQUEST';
+const GET_ROCKETS_SUCCESS = 'space-travellers/rockets/GET_SUCESS';
+const GET_ROCKETS_FAIL = 'space-travellers/rockets/GET_FAIL';
+const RESERVE_ROCKET = 'space-travellers/rockets/RESERVE_ROCKET';
+const UNRESERVE_ROCKET = 'space-travellers/rockets/UNRESERVE_ROCKET';
+
+const initialState = {
+  rocketsData: [],
+  error: '',
+  load: false,
+};
+
+const getRocketRequest = () => ({
+  type: GET_ROCKETS_REQUEST,
+});
+
+const getRocketSuccess = (rocketsData) => ({
+  type: GET_ROCKETS_SUCCESS,
+  payload: rocketsData,
+});
+
+const getRocketFail = (error) => ({
+  type: GET_ROCKETS_FAIL,
+  payload: error,
+});
+
+export function getRockets() {
+  return (dispatch) => {
+    dispatch(getRocketRequest());
+    FetchRocketsData()
+      .then((data) => {
+        const rocketinfo = data.map((uniData) => {
+          const {
+            id,
+            rocket_name: name,
+            description: desc,
+            flickr_image: image,
+          } = uniData;
```

Comment:
I really appreciate the way you assigning to new variable names on object destructing 🏅

Reply:
Thanks Bushra 👍